src/OpenSearch/ConnectionPool/SniffingConnectionPool.php

```diff
@@ -21,30 +21,34 @@
 
 namespace OpenSearch\ConnectionPool;
 
-use OpenSearch\Common\Exceptions\Curl\OperationTimeoutException;
+use Exception;
 use OpenSearch\Common\Exceptions\NoNodesAvailableException;
 use OpenSearch\ConnectionPool\Selectors\SelectorInterface;
 use OpenSearch\Connections\Connection;
-use OpenSearch\Connections\ConnectionInterface;
 use OpenSearch\Connections\ConnectionFactoryInterface;
+use OpenSearch\Connections\ConnectionInterface;
 
-class SniffingConnectionPool extends AbstractConnectionPool implements ConnectionPoolInterface
```

Comment:
This is kinda a breaking change. Can we not keep it? 🤔

Reply:
The parent class `\OpenSearch\ConnectionPool\AbstractConnectionPool` implements the same interface. The implements entry here is redundant or unnecessary. What do you think?

Reply:
ahh okay. didn't saw that with the diff :)

Reply:
Do I need to update the Changelog file, or it is not required for this change?

Reply:
I don't think it's required